frontend/app/components/toolButtons.tsx

```diff
@@ -1,12 +1,7 @@
 import React from 'react';
 import AddIcon from '@mui/icons-material/Add';
 import RemoveIcon from '@mui/icons-material/Remove';
-import { Button, ButtonProps, SvgIconProps, Typography } from '@mui/material';
-
-const iconProps: SvgIconProps = {
-  fontSize: 'medium',
-  sx: { marginRight: 1 }
-};
```

Comment:
I realized that `medium` is the default, and just decided to inline this, since that's how I'm doing it elsewhere. Maybe the icon and text combination with buttons can be extracted some way in the future, but this seems fine for now.

Reply:
I also made this change and the others below in #200, I'll have to rebase depending on what gets merged first.

Reply:
Okay, I rebased and the PR should be limited to the dialog changes now.